pytest_playwright/pytest_playwright.py

```diff
@@ -72,13 +76,6 @@ def pytest_runtest_setup(item: Any) -> None:
         pytest.skip("skipped for this browser: {}".format(browser_name))
 
 
-@pytest.fixture(scope="session")
```

Comment:
`pytest-asyncio` already provides a `loop` fixture

Reply:
Yes I know but I remember that it didn't work without that on some specific project, thats why it was needed here. Need to test.

Reply:
Ok